PlayFabClient.go

```diff
@@ -356,6 +356,7 @@ func GetStoreItems(storeId string, titleId string, playfabId string, catalogVers
 	if !ok {
 		return nil, fmt.Errorf("Failed to parse GetStoreItem result")
 	}
+	storeItems[0].(map[string]interface{})["StoreId"] = data["StoreId"].(string)
```

Comment:
Do you know for sure that the length of storeItems is at least 1?
Why you change only the first item? what is this index-based discrimination? is 0 better than 1?
do you know for sure that `storeItems[]` is a map?  do you know for sure that `data["StoreId"]` indeed exists and is of type string?

what really is your need and how can you achieve it in a more API oriented, robust way?